authentik/root/settings.py

```diff
@@ -18,6 +19,8 @@
 BASE_DIR = Path(__file__).absolute().parent.parent.parent
 STATICFILES_DIRS = [BASE_DIR / Path("web")]
 MEDIA_ROOT = BASE_DIR / Path("media")
+DEFAULT_FILE_STORAGE = "django_tenants.files.storage.TenantFileSystemStorage"
```

Comment:
I wonder if we should consider some other file storage, especially for SaaS, I'd prefer not having to have a PVC mounted on all these pods

Reply:
Yes, my thought was to add optional support for S3 in another PR. I think this PR is large enough as it is. We can also decide to keep the default filesystem storage as it is currently, and force the use of S3 when using tenants.

Reply:
Yeah definitely makes sense to do that in another PR. As much as I don't want to support multiple ways of doing something, we'd probably keep the local file storage support around for compose installs, and maybe change the current other method (of just inputting URLs) to use S3

Reply:
I actually reverted the revert, because I think we still need to keep tenants files separated even if on disk.

Reply:
Okay, so a couple of modifications had to be made to the filesystem backend as well as django-tenants doesn't support django 5 for this.
I merged #7861 here to see what it would look like with tenants.

Reply:
Ah nvm github won't let me push anymore